cogs/char.py

```diff
@@ -3178,15 +3196,30 @@ async def image(self,ctx, char, url):
                 print('Success')
                 await ctx.channel.send(content=f'I have updated the image for ***{char}***. Please double-check using one of the following commands:\n```yaml\n{commandPrefix}info "character name"\n{commandPrefix}char "character name"\n{commandPrefix}i "character name"```')
     
+    @commands.cooldown(1, 5, type=commands.BucketType.member)
+    @is_log_channel()
+    @commands.has_any_role('A d m i n')
+    @commands.command(aliases=['wop'])
+    async def updatedatabase(self, ctx): #moves the Reflavor value in every character that has one to a new Reflavor dictionary under New Race
+
+        for line in db.players.find({ "Reflavor": { "$exists": 'true' } }):
+            db.players.update_one({"Reflavor": line["Reflavor"]}, [{"$set": {"Reflavor": {"race": line["Reflavor"]}}}])
+            
+        await ctx.channel.send(content="You have changed the data type for Reflavor. DO NOT USE THIS COMMAND EVER AGAIN.")
+        
+    
     @commands.cooldown(1, 5, type=commands.BucketType.member)
     @is_log_channel()
     @commands.command(aliases=['rf'])
-    async def reflavor(self,ctx, char, *, new_race):
-        if( len(new_race) > 20 or len(new_race) <1):
+    async def reflavor(self,ctx, rtype, char, *, new_flavor):
+        if( len(new_flavor) > 20 or len(new_flavor) <1):
             await ctx.channel.send(content=f'The new race must be between 1 and 20 symbols.')
             return
-
-    
+        rtype = rtype.lower()      
+        if(rtype != "race" and rtype != "background" and rtype != "class"): #makes sure that the user put in a valid reflavor type, in a messy way that doesn't fit the format of our other error messages.
+            await ctx.channel.send(content=f'***{rtype}*** is an invalid reflavor type.')
+            return
+            
         channel = ctx.channel
```

Comment:
Instead of limiting it to only 3 possible inputs make 3 command linking to a new function. That will also allow us to easily keep the formatting standard for keys in the DB (first letter capitalized)

Reply:
Would you suggest the command names be $race $class and $background respectively?

Reply:
$reflavor race etc...
make $reflavor a group